samples/excel/85-preview-apis/range-direct-dependents.yaml

```diff
@@ -0,0 +1,186 @@
+order: 4
+id: excel-direct-dependents
```

Comment:
Since these APIs are in preview, a sample separate from the precedents API makes sense. We should consider merging them down the line, since the APIs are closely related and should probably be discovered at the same time.

Reply:
I agree.